terraform/monitoring/04_alerting_policies.tf

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Here we create alerting policies on the Service SLO for the frontend service
+# The definition of the SLO can be found in the file "03_service_slo.tf"
+# We alert on budget burn rate, alerting if burn rate exceeds twice the allotted burn rate
+# within the past hour 
+resource "google_monitoring_alert_policy" "frontend_availability_slo_alert" {
+  display_name = "Frontend Service Availability Alert Policy"
+  combiner     = "AND"
+  conditions {
+    display_name = "SLO burn rate alert for availability SLO with a threshold of 2"
+    condition_threshold {
+
+      # This filter alerts on burn rate over the past 60 minutes
+      filter     = "select_slo_burn_rate(\"projects/${var.project_id}/services/${google_monitoring_custom_service.frontend.service_id}/serviceLevelObjectives/${google_monitoring_slo.frontend_availability_slo.slo_id}\", 60m)"
+      
+      threshold_value = "2"
+      comparison = "COMPARISON_GT"
+    }
+  }
+  documentation {
+    content = "SLO burn for the past 60m exceeded twice the acceptable budget burn rate."
```

Comment:
Typically these documentation pages are called TSGs (troubleshooting guides) and their goal is to provide some hints on potential next steps to mitigate the issue. Please add a bit more details on what the alert means and what next steps could be.

Reply:
I added more details - let me know if you think these are sufficient. I tried to describe a) what the issue was that was triggering, and b) where they can look for information/next steps